src/lib/server/Server.cpp

```diff
@@ -163,6 +163,10 @@ Server::Server(
 							m_inputFilter,
 							new TMethodEventJob<Server>(this,
 								&Server::handleSwitchToScreenEvent));
+	m_events->adoptHandler(m_events->forServer().toggleScreen(),
```

Comment:
Can we please use spaces for new code? Different viewers will indent tabs differently and this code looks weird e.g. on GitHub.

Reply:
I don't like tabs either, but this file use tabs. To match the whole file formatting, I use tab here.

Reply:
If the rest of the [Barrier] code uses tabs, we should stick with tabs. Or we can reformat all code to use a unified spacing - either way, we should use one or another to keep it universal.

Reply:
Given that even upstream is switching code bit by bit to spaces, I think we should just use spaces. Tabs are just painful to work with as there's always some tool that does not support it. I worked at a 500 software engineer company that couldn't make tabs work due to that reason and eventually switched everything to spaces.

For us the most sensible solution would be to just use spaces in new code. Converting everything would introduce unnecessary merge conflicts with upstream and will make `git blame` unusable.

Reply:
E.g. 207247b7efe143255d4b57784f67852d55666eb6.

Reply:
Alright, let's use spaces in new code then. Ideally converting all of it to spaces would be great, but I think you're right in "keeping a balance" :+!:

Reply:
Although I'd be keen to convert tabs to spaces.. I can't see any other way to keep `git blame` working..

Reply:
I think using spaces only for new code shouldn't be a problem. Most editors support setting whitespace settings for modified lines only. The overhead of not committing extra changes accidentally should also be low, because a developer should review his changes before commit anyway.

Reply:
Fair enough :smile:

Reply:
using space now